content/_help/verify-your-identity/how-to-verify-your-identity._en.md

```diff
@@ -20,12 +20,12 @@ do.
 ## Requirements for identity verification
 
 {: .number-list}
-1. Your [drivers license or state ID card.](/help/verify-your-identity/accepted-identification-documents/) You can upload a photo of your ID by phone or by computer.
-1. Social Security number
-1. Your phone number 
-    * If we can't verify your phone number, you can [verify by mail](/help/verify-your-identity/verify-your-address-by-mail/) instead. This takes 5 to 10 days.
-1. Taking a photo of yourself
-Some services need you to take a photo of yourself with a phone to verify your identity online. If you don't have a phone with a camera, you can [verify your identity in person.](/help/verify-your-identity/verify-your-identity-in-person/)
+1. Your [driver's license or state ID card.](/help/verify-your-identity/accepted-identification-documents/) You can upload a photo of your ID by phone or by computer.
+2. Social Security number
```

Comment:
This makes the diff messy, unfortunately. I normally don't care about whether Markdown numbered lists repeat 1 or go in order, and might even argue that using 1. for each item is better in case you need to change numbering.

*But*, with translations, it's very helpful to be able to know that I'm replacing the text on item number 5 vs. trying to bumble my way through the text.

Reply:
With translations are we submitting articles piecemeal or are we submitting the whole article?

Reply:
Piecemeal. In this case we [just had some strings](https://docs.google.com/spreadsheets/d/1w5iktBb7OZCuplF0M2dIT0Use9fkYnRmLcLXJ3NHmcc/edit?gid=0#gid=0) in a spreadsheet.

Reply:
My understanding is translators usually like the full context of the whole article, so copy-pasting the whole thing might make for an easier round-trip in translations as well 🤷